analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -64,7 +72,7 @@ public void Dispose_Stream_LeaveOpenTrue()
 
     public void Disposed_Using_WithDeclaration()
     {
-        using (var d = new Disposable()) // FIXME Non-compliant
+        using (var d = new Disposable()) // Noncompliant
```

Comment:
Can we raise an issue on a previous state?
I think here would be better to raise in the dispose and not in using.

Reply:
MMF-2568 should introduce displaying flows. So the other place will be a secondary location. So in the future, we will be able to do that.